### ISSUE TITLE
LED call & wait

### ISSUE THREAD

**Issue by Chris-ParkIT** [hthiery/python-lacrosse#7]
I am trying to run this on a windows machine, I have got the py code rather than installed the package

firstly in clitool line 91 the call to  lacrosse.led_config(state)
should be to led_mode_state(state)
the command to set the blue led should be 1l (lowercase L) not 1a

if I run with parameters `-d /COM3 led on`  nothing happens
putting a `time.sleep(3)` at the start of led_mode_state() the blue light comes on

If I run `-d /COM3 info` the data returned is 'b\r\n' and so all info shows as nothing
running '-d /COM3 scan' nothing gets returned.
I assume it is a timing issue but don't have the experience or knowledge to get any further.

**Comment by hthiery** [comment]
> I am trying to run this on a windows machine, I have got the py code rather than installed the package
> 
> firstly in clitool line 91 the call to lacrosse.led_config(state)
> should be to led_mode_state(state)
> the command to set the blue led should be 1l (lowercase L) not 1a

You're right. this was never tested. could you please create a pull-request for that?

> if I run with parameters `-d /COM3 led on` nothing happens
> putting a `time.sleep(3)` at the start of led_mode_state() the blue light comes on
>
> If I run `-d /COM3 info` the data returned is 'b\r\n' and so all info shows as nothing
> running '-d /COM3 scan' nothing gets returned.
> I assume it is a timing issue but don't have the experience or knowledge to get any further.

what firmware version of jeelink dongle do you use?

**Comment by Chris-ParkIT** [comment]
Thanks for the quick response, 
logging 'print(line)' for the info request gives a response:
b'  <n> b      - set MHz band (4 = 433, 8 = 868, 9 = 915)\r\n'
b'  <nnnn> o   - change frequency offset within the band (default 1600)\r\n'
b'               96..3903 is the range supported by the RFM12B\r\n'
b'  <nnn> g    - set network group (RFM12 only allows 212, 0 = any)\r\n'
b'  <n> c      - set collect mode (advanced, normally 0)\r\n'
b'  t          - broadcast max-size test packet, request ack\r\n'
b'  ...,<nn> a - send data packet to node <nn>, request ack\r\n'
b'  ...,<nn> s - send data packet to node <nn>, no ack\r\n'
b"  <n> q      - set quiet mode (1 = don't report bad packets)\r\n"
b'  <n> x      - set reporting format (0: decimal, 1: hex, 2: hex+ascii)\r\n'
b'  123 z      - total power down, needs a reset to start up again\r\n'
b'Remote control commands:\r\n'
b'  <hchi>,<hclo>,<addr>,<cmd> f     - FS20 command (868 MHz)\r\n'
b'  <addr>,<dev>,<on> k              - KAKU command (433 MHz)\r\n'
b'Current configuration:\r\n'
b' A i1 g210 @ 868 MHz q1\r\n'
b'> 0v\r\n'
b'[RF12demo.12] A i1 g210 @ 868 MHz q1\r\n'
which the regexp isn't matching...
I cannot see a firmware version - is there a way of getting this?  I have only just bought it so assume it is a very recent one then.
